tools/bin/hawqregister

```diff
@@ -397,6 +397,14 @@ class HawqRegister(object):
                     logger.info('%s is not a file in hdfs, please check the yaml configuration file.' % fn)
                     sys.exit(1)
 
+        def check_sizes_valid():
+            for k, fn in enumerate(self.files):
+                hdfscmd = 'hdfs dfs -du %s' % fn
+                _, out, _ = local_ssh_output(hdfscmd)
+                if self.sizes[k] > int(out.strip().split()[0]):
+                    logger.error('Incorrect filesize %s in yaml of file %s: out of range %s' % (self.sizes[k], fn, out.strip().split()[0]))
```

Comment:
File size(%s) in yaml configuration file should not exceed actual length(%s) of file %s